### DISCUSSION TITLE
WebRTC with UWP

### DISCUSSION THREAD

**Discussion by wtjohnson** [sipsorcery-org/sipsorcery#1025] [General]
Could I get some guidance on how to use your WebRTC Features with UWP, specifically the WebViewer?

**Comment by sipsorcery** [comment]
I've never tried it but the main library should work fine in a UWP app as it does not have any OS dependencies. If you only need WebRTC datachannels it could be a good fit. 

The issue will be if you need to use audio or video devices or need a native video codec.
